buildlib/azure-pipelines.yml

```diff
@@ -1,20 +1,53 @@
 # See https://aka.ms/yaml
 # This pipeline to be run on direct pushes and merges
 
-pr: none
 trigger:
   - master
   - v*.*.x
 
+pr:
+  - master
+  - v*.*.x
```

Comment:
this is temp?

Reply:
For check pipeline on PR, can remove or or not
We check it only if PR have prefix `AZP/SNAPSHOT: `